packages/next/src/elements/Nav/index.client.tsx

```diff
@@ -61,7 +61,14 @@ export const DefaultNavClient: React.FC<{
               }
 
               return (
-                <Link className={`${baseClass}__link`} href={href} id={id} key={i} prefetch={false}>
+                <Link
+                  className={`${baseClass}__link ${isActive && 'active'}`}
```

Comment:
This might cause `false` to be included in the classnames. Should just use the filter/join trick here.

Reply:
Honestly there is no need for this class. Should just use css `:has(.nav__link-indicator)` and apply bold from there.